spec/features/work_packages/details/relations/primerized_relations_spec.rb

```diff
@@ -251,6 +252,49 @@ def label_for_relation_type(relation_type)
       # Bumped by one
       tabs.expect_counter("relations", 5)
     end
+
+    it "doesn't autocomplete parent, children, and WP itself" do
+      scroll_to_element relations_panel
+
+      relations_panel.find("[data-test-selector='new-relation-action-menu']").click
+
+      within page.find_by_id("new-relation-action-menu-list") do
+        click_link_or_button "Child"
+      end
```

Comment:
Please extract that into a method. The helper already has other places where the create dialog is opened and where the method then could be used as well.